fluent/protocol/transport.go

```diff
@@ -40,11 +42,15 @@ func init() {
 	})
 
 	compressorPool.New = func() interface{} {
-		return &GzipCompressor{}
+		return new(GzipCompressor)
```

Comment:
Just out of curiosity: what is the reason to switch to new() here instead of &{}?